tests/control_flow_test.py

```diff
@@ -329,27 +335,8 @@ def test_fsm():
 
     if dace.Config.get_bool('optimizer', 'detect_control_flow'):
         code = sdfg.generate_code()[0].clean_code
-        assert 'switch ' in code
-
-
-def test_nested_loop_detection():
```

Comment:
This is redundant since this kind of construct is now explicit control flow straight from the frontend (tested there), and there are separate loop detection tests that cover nested loops for control flow raising.